### DISCUSSION TITLE
Issue about the dateset management command of USB to NVME device

### DISCUSSION THREAD

**Discussion by hxh202** [Seagate/cortx#1595] [Q&A]
Hello, I'd like to ask whether the unmap command of SCSI can be used to send the dateset management command to USB to NVME devices in Windows？ I only that the NVME device is OK in MSDN, and I don't know whether it is feasible to connect to the USB interface

**Comment by hessio** [comment]
@johnbent do you know?

**Comment by johnbent** [comment]
I don't know. This is a low-level hardware question. Is this even somehow related to CORTX? :)

**Reply by hessio** [reply]
Ya I don't think this is related to CORTX, @hxh202 can you clarify what your issue, I'm a little unclear on what you are asking?

**Comment by hxh202** [comment]
thanks for your reply. this problem has been solved.nvm command may not be supported in windows.now use scsi unmap command is ok

------------------ Original ------------------
From: John Bent ***@***.***&gt;
Date: Fri,Jul 8,2022 2:40 AM
To: Seagate/cortx ***@***.***&gt;
Cc: Aurora ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [Seagate/cortx] Issue about the dateset management command of USB to NVME device (Discussion #1595)





 
I don't know. This is a low-level hardware question. Is this even somehow related to CORTX? :)
